proposals/wac-ucr/index.bs

```diff
@@ -1114,13 +1114,178 @@ and the presentation goes off perfectly.
 Requirements {#requirements}
 ================================================================================
 
-Issue: Populate requirements based on established use cases.
+## Functional Requirements ## {#req-functional}
+
+1. The system shall permit or restrict an [=agent's=] ability to access a 
+    [=resource=] or [=collection=]. [[#uc-basic]], 
+    [[#uc-collections]], [[#uc-blockagents]]
```

Comment:
On its own I agree that it is general, but as the first requirement it is meant to set a context for the requirements that follow it. Specifically, by setting the context that the system permits or restricts an agent's ability to _access a resource or collection_ in the first requirement, and linking to the major sections, we could avoid having to repeat "_access to a resource or collection_" in almost all of the others.

Reply:
Each functional requirement should unambiguously describe what's needed so that possible solutions can be proposed. We're not going to come up with "a" solution for this requirement that'd be different than what would be covered by fine-grained requirements. It is redundant. Section description sets the context.

Reply:
Will replace with an introduction that sets some context.

Reply:
Note added to introduction at https://github.com/solid/authorization-panel/pull/152/commits/dcf9b4fd6febecb57865c4c6f647fd6e28aae207